File: apk/app/src/main/java/com/example/momentra/ui/shell/group/wedding/WeddingDemoContent.kt
package com.example.momentra.ui.shell.group.wedding

/**
 * QUARANTINED — static Figma copy only. Must never be wired into production UI paths.
 * Live Wedding screens use APIs + honest empties. Kept for isolated previews/tests.
 */
@Deprecated("Do not use in production UI — quarantine for Figma layout reference only")
object WeddingDemoContent {
    const val HEALTH_SCORE = 82
    const val HEALTH_SUBTITLE = "Coordination is moving smoothly."
    const val HEALTH_DETAIL = "Planning momentum is strong. Keep the team aligned!"
    const val HEALTH_UPDATED = "Updated 2 hours ago"
    const val HERO_DATE_LABEL = "18 Oct 2026"
    const val HERO_COUNTDOWN = "2 months away!"
    const val HERO_TYPE_PILL = "Wedding • 18 Oct 2026"

    data class StatusPill(val label: String, val emoji: String)
    val healthPills = listOf(
        StatusPill("74% Activity", "💖"),
        StatusPill("8 Replies", "👥"),
        StatusPill("Trending Up", "✨"),
    )

    data class AttentionItem(val emoji: String, val title: String, val detail: String, val cta: String)
    val attentionItems = listOf(
        AttentionItem("🏛️", "Venue deposit due", "Finalize payment by Oct 1", "Pay Now"),
        AttentionItem("📋", "Guest list finalization", "Confirm headcount for catering", "Review"),
        AttentionItem("🍽️", "Catering menu selection", "Finalize appetizers and desserts", "Choose"),
    )

    const val PROGRESS_PERCENT = 75
    const val TASKS_DONE = 12
    const val TASKS_TOTAL = 16
    const val DEMO_RSVP_COUNT = "74"
    const val DEMO_RSVP_PERCENT = "87%"
    const val DEMO_TASKS = "18"
    const val SEGMENT_FILLED = 6

    data class PartyMember(val name: String, val role: String, val percent: Int, val featured: Boolean = false)
    val partyMembers = listOf(
        PartyMember("Sarah", "Bride", 92, featured = true),
        PartyMember("Mike", "Groom", 78),
        PartyMember("Priya", "Maid of Honor", 72),
        PartyMember("Arjun", "Best Man", 48),
        PartyMember("Meera", "Planner", 55),
    )

    data class BudgetCategory(val label: String, val amountLabel: String, val fraction: Float)
    val budgetCategories = listOf(
        BudgetCategory("Venue & Catering", "₹2,80,000", 0.85f),
        BudgetCategory("Decor & Photography", "₹1,20,000", 0.45f),
    )
    const val DEMO_BALANCE_OWE = "You owe ₹15,000"
    const val DEMO_TOTAL_POOL = "₹5,20,000"

    data class ActivityDemo(val emoji: String, val title: String, val whenLabel: String)
    val activityFeed = listOf(
        ActivityDemo("📸", "Sarah added venue photos", "Today, 9:42 AM"),
        ActivityDemo("🎵", "Mike confirmed DJ booking", "Today, 8:15 AM"),
        ActivityDemo("👗", "Priya shared outfit ideas", "Yesterday, 11:23 PM"),
        ActivityDemo("🎁", "Meera added registry items", "Yesterday, 6:00 PM"),
    )

    data class Insight(val emoji: String, val title: String, val body: String)
    val insights = listOf(
        Insight("👥", "Guest coordination", "Guest coordination is 85% complete — Finalize RSVP this week."),
        Insight("📷", "Venue momentum", "Venue photos boosted planning momentum — Share more venue details to keep it up."),
    )

    data class TimelineDay(val title: String, val date: String, val detail: String, val time: String, val emoji: String)
    val momentsTimeline = listOf(
        TimelineDay("MEHNDI", "17 OCT", "Mehndi ceremony & cocktails", "4:00 PM", "☀️"),
        TimelineDay("WEDDING", "18 OCT", "Main ceremony & reception", "11:00 AM", "🛕"),
        TimelineDay("RECEPTION", "18 OCT", "Grand reception & dinner", "7:00 PM", "🌙"),
    )

    data class GalleryTile(val title: String, val countLabel: String? = null)
    val galleryTiles = listOf(
        GalleryTile("Venue decor", "12"),
        GalleryTile("Mehndi setup", "8"),
        GalleryTile("Outfits", "6"),
    )

    data class UpcomingEvent(val title: String, val detail: String, val badge: String?, val emoji: String)
    val upcomingEvents = listOf(
        UpcomingEvent("Venue walkthrough", "Final check • 12 Oct", "THIS SATURDAY", "🏠"),
        UpcomingEvent("Outfit fitting", "Next week • Bride & bridesmaids", null, "📅"),
    )

    data class MemoryMilestone(val title: String, val date: String, val emoji: String)
    val memoryTimeline = listOf(
        MemoryMilestone("The group was created", "02 Aug 2026", "⭐"),
        MemoryMilestone("Venue confirmed and booked", "18 Sep 2026", "📍"),
        MemoryMilestone("First dress fitting captured", "03 Oct 2026", "👜"),
        MemoryMilestone("Mehndi night memories", "13 Dec 2026", "📷"),
    )

    val memoryFeatureCards = listOf(
        "First plan confirmed",
        "Everyone contributed",
        "50 RSVPs received",
    )

    val memoryInsights = listOf(
        Pair("Strongest memory signal", "Mehndi night was the most emotional shared moment."),
        Pair("Participation pattern", "Photos were captured by all active members."),
    )
}

File: momentra/momentra/Shell/GroupActive/Wedding/Create/WeddingDemoContent.swift
import SwiftUI

/// QUARANTINED — static Figma demo copy only. Never wire into production UI paths.
/// Live Wedding screens use APIs + honest empties. Kept for isolated previews/tests.
@available(*, deprecated, message: "Do not use in production UI — quarantine for Figma layout reference only")
enum WeddingDemoContent {
    static let healthScore = 82
    static let healthSubtitle = "Coordination is moving smoothly."
    static let healthDetail = "Planning momentum is strong. Keep the team aligned!"
    static let healthUpdated = "Updated 2 hours ago"
    static let heroDateLabel = "18 Oct 2026"
    static let heroCountdown = "🎉 2 months away!"
    static let heroTypePill = "Wedding • 18 Oct 2026"

    static let healthPills = ["74% Activity", "8 Replies", "Trending Up"]
    
    static let segmentFilled = 6
    static let segmentTotal = 8
    static let demoRsvpCount = "74"
    static let demoTasksDone = "12"
    static let demoTasksTotal = "16"

    struct AttentionItem {
        let emoji: String
        let title: String
        let detail: String
        let cta: String
    }

    static let attentionItems: [AttentionItem] = [
        .init(emoji: "💰", title: "Venue deposit due", detail: "Finalize payment by Oct 1", cta: "Pay Now"),
        .init(emoji: "👥", title: "Guest list finalization", detail: "Confirm headcount for catering", cta: "Review"),
        .init(emoji: "🍽️", title: "Catering menu selection", detail: "Finalize appetizers and desserts", cta: "Choose"),
    ]

    static let progressPercent = 75
    static let tasksDone = 12
    static let tasksTotal = 16
    static let demoRsvpPercent = "87%"
    static let demoTasks = "18"

    struct PartyMember {
        let name: String
        let role: String
        let percent: Int
        let featured: Bool
    }

    static let partyMembers: [PartyMember] = [
        .init(name: "Sarah", role: "Bride", percent: 92, featured: true),
        .init(name: "Mike", role: "Groom", percent: 78, featured: false),
        .init(name: "Priya", role: "Maid of Honor", percent: 72, featured: false),
        .init(name: "Arjun", role: "Best Man", percent: 48, featured: false),
        .init(name: "Meera", role: "Planner", percent: 55, featured: false),
    ]

    struct BudgetCategory {
        let label: String
        let amountLabel: String
        let fraction: CGFloat
    }

    static let budgetCategories: [BudgetCategory] = [
        .init(label: "Venue & Catering", amountLabel: "₹2,80,000", fraction: 0.85),
        .init(label: "Decor & Photography", amountLabel: "₹1,20,000", fraction: 0.45),
    ]

    struct ActivityDemo {
        let emoji: String
        let title: String
        let whenLabel: String
    }

    static let activityFeed: [ActivityDemo] = [
        .init(emoji: "📷", title: "Sarah added venue photos", whenLabel: "Today, 9:42 AM"),
        .init(emoji: "🎵", title: "Mike confirmed DJ booking", whenLabel: "Today, 8:15 AM"),
        .init(emoji: "👗", title: "Priya shared outfit ideas", whenLabel: "Yesterday, 11:23 PM"),
        .init(emoji: "🎁", title: "Meera added registry items", whenLabel: "Yesterday, 6:00 PM"),
    ]

    static let insights: [(emoji: String, title: String, body: String)] = [
        (emoji: "✨", title: "Guest coordination", body: "85% complete — Finalize RSVP this week."),
        (emoji: "📸", title: "Venue momentum", body: "Photos boosted planning momentum — Share more venue details to keep it up."),
    ]

    struct TimelineDay {
        let emoji: String
        let title: String
        let date: String
        let detail: String
        let time: String
    }

    static let momentsTimeline: [TimelineDay] = [
        .init(emoji: "🎨", title: "MEHNDI", date: "17 OCT", detail: "Mehndi ceremony & cocktails", time: "4:00 PM"),
        .init(emoji: "💒", title: "WEDDING", date: "18 OCT", detail: "Main ceremony & reception", time: "11:00 AM"),
        .init(emoji: "🥂", title: "RECEPTION", date: "18 OCT", detail: "Grand reception & dinner", time: "7:00 PM"),
    ]

    struct GalleryTile {
        let title: String
        let countLabel: String?
    }

    static let galleryTiles: [GalleryTile] = [
        .init(title: "Venue decor", countLabel: "12"),
        .init(title: "Mehndi setup", countLabel: "8"),
        .init(title: "Outfits", countLabel: "6"),
    ]

    struct UpcomingEvent {
        let title: String
        let detail: String
        let badge: String?
    }

    static let upcomingEvents: [UpcomingEvent] = [
        .init(title: "Venue walkthrough", detail: "Final check • 12 Oct", badge: "THIS SATURDAY"),
        .init(title: "Outfit fitting", detail: "Next week • Bride & bridesmaids", badge: nil),
    ]

    struct MemoryMilestone {
        let title: String
        let date: String
    }

    static let memoryTimeline: [MemoryMilestone] = [
        .init(title: "The group was created", date: "02 Aug 2026"),
        .init(title: "Venue confirmed and booked", date: "18 Sep 2026"),
        .init(title: "First dress fitting captured", date: "03 Oct 2026"),
        .init(title: "Mehndi night memories", date: "13 Dec 2026"),
    ]

    static let memoryFeatureCards = [
        "First plan confirmed",
        "Everyone contributed",
        "50 RSVPs received",
    ]

    static let memoryInsights: [(String, String)] = [
        ("Strongest memory signal", "Mehndi night was the most emotional shared moment."),
        ("Participation pattern", "Photos were captured by all active members."),
    ]
}
